run.py

```diff
@@ -475,6 +477,7 @@ def fetch_test_details(var) -> dict:
         details["name"] = var.get("name")
         details["desc"] = var.get("desc")
         details["file"] = var.get("module")
+        details["cli_used"] = cli_used
```

Comment:
Likewise, can we change the key... I believe it would be easily understood the expected values.